plugin/trino-phoenix5/src/main/java/io/trino/plugin/phoenix5/PhoenixClientModule.java

```diff
@@ -209,6 +212,10 @@ public static Properties getConnectionProperties(PhoenixConfig config)
 
         ConnectionInfo connectionInfo = ConnectionInfo.create(config.getConnectionUrl(), EMPTY_PROPS, new Properties());
         connectionInfo.asProps().asMap().forEach(connectionProperties::setProperty);
+        Duration serverScanPageTimeout = config.getServerScanPageTimeout();
+        if (!serverScanPageTimeout.equals(DEFAULT_SERVER_SCAN_PAGE_TIMEOUT)) {
```

Comment:
Consider using `Optional<Duration>` instead in PhoenixConfig.

Reply:
You mean we return Optional.empty() from PhoenixConfig if the value is default i.e. `Long.MAX_VALUE`?